### ISSUE TITLE
Failed at the linaria@0.2.0 postinstall script

### ISSUE THREAD

**Issue by pcxiong** [callstack/linaria#158]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
npm install failed
```
sh: yarn: command not found
...
npm ERR! linaria@0.2.0 postinstall: `yarn link && yarn link linaria`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the linaria@0.2.0 postinstall script.
```

**If the current behavior is a bug, please provide the steps to reproduce and a minimal repository on GitHub that we can `yarn install` and `yarn test`.**
`npm install --save linaria`

**What is the expected behavior?**
correctly be installed

**Please provide your exact Babel configuration and mention your Linaria, Node, Yarn/npm version and operating system.**
Linaria 0.2.0 / node 8.6.0 / npm 5.5.1 / macOS 10.12.6

**Comment by satya164** [comment]
cc @thymikee

**Comment by thymikee** [comment]
Oh, looks like npm is running postinstall scripts of the installed packages, while Yarn is not. I'll take care of it later this day.

**Comment by thymikee** [comment]
Fixed in #167
